client/client.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+  <title>Pusher Test</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+  <script src="https://js.pusher.com/4.3/pusher.min.js"></script>
+  <script>
+
+    // Enable pusher logging - don't include this in production
+    Pusher.logToConsole = true;
+
+    var pusher = new Pusher('672d4a2dcf394b52247d', {
```

Comment:
The string you initialized the `Pusher` class with, where is it from?

Reply:
It is the user.id. By default, it is meant to come from decoded token in frontend.

Reply:
So whose ID is the one currently hardcoded here?

Reply:
It was the user.id from user table in my database.